bin/dipy_median_ostu

```diff
@@ -0,0 +1,18 @@
+#! /usr/bin/env python
+
+from dipy.workflows.segment import median_otsu_bet
+from dipy.fixes import argparse as arg
+
+parser = arg.ArgumentParser()
+parser.add_argument('input', action='store')
+parser.add_argument('--out_dir', type=str, default='')
+parser.add_argument('--save_masked', action='store_true')
+parser.set_defaults(save_masked=False)
+parser.add_argument('--radius', type=int, default=4)
```

Comment:
This is nice. Apart from the missing documentation/help/example. 
But here I am suggesting a radical change which I believe will save us from concentrating on the cmds and ArgumentParsers and work more on the actual workflow. So, my suggestion is that we should assume that all inputs of workflows are always strings and therefore we will cast the needer parameters inside the workflow.  I am still playing in my head with this idea but I think it will simplify many things in the relative close to far future. 

So, no casting will happen in the ArgumentParser too. All inputs are strings and are given to the relevant workflow.
@arokem, @matthieudumont  can you think a bit this suggestion? Are the advantages clear or do I need to elaborate more?